### PR TITLE
feat: add address command

### PR DESCRIPTION
**Pull request by aboutlo**
Added the get address from the private key provided

`npx . address --private-key ${PRIVATE_KEY}`

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 
-on:
-  push:
-    branches:
-      - '!master'
+#on:
+#  push:
+#    branches: '!master'
+
+on: [push]
```
Comment (aboutlo) [inline .github/workflows/ci.yaml:7]: I don't get why this prevent the `ci` workflow to start

```
on:
  push:
    branches:
      - '!master'
```

Comment (aboutlo) [reply .github/workflows/ci.yaml]: Solved with `branches-ignore`

**Review by Scott-Canning — APPROVED** [review]
LGTM

**Pull request merged**